instance/models/instance.py

```diff
@@ -86,7 +87,8 @@ class Instance(ValidateModelMixin, TimeStampedModel):
     STARTED = 'started'
     ACTIVE = 'active'
     BOOTED = 'booted'
-    PROVISIONED = 'provisioned'
+    PROVISIONING = 'provisioning'
+    ERROR = 'error'
```

Comment:
@chromano You don't need this state now that you have `PROGRESS_FAILED`? I don't think your code is using this state anymore.